src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Employee/EmployeeOptionsFormDataProvider.php

```diff
@@ -60,6 +71,33 @@ public function getData()
      */
     public function setData(array $data)
     {
+        if ($errors = $this->validate($data)) {
+            return $errors;
+        }
+
         return $this->employeeOptionsConfiguration->updateConfiguration($data);
```

Comment:
following comment above we can simplify it to even this:
```suggestion
        return $this->validate($data) ?: $this->employeeOptionsConfiguration->updateConfiguration($data);
```

Reply:
It looks better ;-) Thx